Ports/dmidecode/package.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=dmidecode
+version=3.3
+useconfigure=false
+prefix=
+files="http://download.savannah.gnu.org/releases/dmidecode/dmidecode-${version}.tar.xz dmidecode-${version}.tar.xz"
```

Comment:
The archive file should be validated using the provided signature. The signature file (preferably both the archive and signature) should also be downloaded over HTTPS.

Untested, but this should do the trick:

```suggestion
files="https://download.savannah.gnu.org/releases/dmidecode/dmidecode-${version}.tar.xz dmidecode-${version}.tar.xz
https://download.savannah.gnu.org/releases/dmidecode/dmidecode-${version}.tar.xz.sig dmidecode-${version}.tar.xz.sig"
auth_type="sig"
auth_opts="--keyring ./gnu-keyring.gpg dmidecode-${version}.tar.xz.sig"
```